app/Filament/Resources/MetricResource.php

```diff
@@ -175,34 +177,35 @@ public static function form(Form $form): Form
-                                            ->required()
-                                            ->relationship('dimension', 'name'),
-                                        Select::make('parent_id')
-                                            ->relationship('parent', 'name')
-                                            ->label('Is this is a descendant of another sub-dimension?'),
-                                        TextInput::make('name'),
-                                        Textarea::make('notes'),
-                                    ])
-                                    ->multiple()
-                                    ->createOptionAction(fn (Action $action): Action => $action->tooltip('Create new'))
-                                    ->suffixAction(self::makeDiscussionPointAction()),
+                                // do not show sub-dimentsions in front end
+                                // Placeholder::make('Sub Dimensions')
+                                //     ->label('0.e. Sub-dimensions')
+                                //     ->content('Select one or more Dimensions before selecting sub-dimensions')
+                                //     ->hidden(fn (callable $get) => $get('dimensions') !== []),
+
+                                // Select::make('subDimensions')
+                                //     ->label('0.e. Sub-dimensions')
+                                //     ->hidden(fn (callable $get) => $get('dimensions') === [])
+                                //     ->relationship('subDimensions', 'name')
+                                //     ->options(
+                                //         fn (callable $get) => SubDimension::whereIn('dimension_id', $get('dimensions'))
+                                //             ->orderBy('dimension_id')
+                                //             ->get()
+                                //             ->pluck('name', 'id')->toArray()
+                                //     )
+                                //     ->createOptionForm([
+                                //         Select::make('dimension_id')
+                                //             ->required()
+                                //             ->relationship('dimension', 'name'),
+                                //         Select::make('parent_id')
+                                //             ->relationship('parent', 'name')
+                                //             ->label('Is this is a descendant of another sub-dimension?'),
+                                //         TextInput::make('name'),
+                                //         Textarea::make('notes'),
+                                //     ])
+                                //     ->multiple()
+                                //     ->createOptionAction(fn (Action $action): Action => $action->tooltip('Create new'))
+                                //     ->suffixAction(self::makeDiscussionPointAction()),
```

Comment:
Let's completely delete this - it's in the git history if we ever do need to go back.